gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -11,6 +11,8 @@ let webpackManifest = require('./webpackManifest')
 let _object = require('lodash/object');
 
 module.exports = function (env) {
+  process.env['BABEL_ENV'] = process.env['BABEL_ENV'] || process.env['NODE_ENV'] || env
```

Comment:
I had started with simply `process.env['BABEL_ENV'] = env` but decided that if it were already set to use the pre-defined value.

Not sure for `NODE_ENV` however. What do you think?